bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -137,7 +144,6 @@ public void run() {
       activeMutationWorkers.incrementAndGet();
       try {
         while (!executorService.isShutdown()) {
-          Mutation mutation = null;
```

Comment:
it turned out to be superfluous.  issueRequest() handles the Exception case already.  This logic will never be invoked.